src/core/components/ChatInput.tsx

```diff
@@ -37,16 +39,16 @@ const ChatInput = ({ setShouldAutoScroll }: ChatInputProps) => {
 
   const [inputValue, setInputValue] = useState('');
 
-  const [dragState, setDragState] = useState({
-    isDragging: false,
+  const [uploadingState, setUploadingState] = useState({
+    isActive: false,
```

Comment:
Renamed this state variable & property since we are now leveraging it for "active uploading state" whether through icon click or drag & drop.